### ISSUE TITLE
Rule "Remove space before commas and period" duplicates template comment in function module 

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#348]
Hi Jörg-Michael, the rule "Remove space before commas and period" causes a problem in function modules without parameters.

Example:

```abap
FUNCTION z_test.
  WRITE 'hello world'.
ENDFUNCTION.
```

When this function module is saved, a comment is automatically added by SAP standard:

```abap
FUNCTION Z_TEST
 " You can use the template 'functionModuleParameter' to add here the signature!
.

  WRITE 'hello world'.
ENDFUNCTION.
```

Ignoring the infuriating grammar issue in the comment, this is how it looks like after the ABAP Cleaner is executed:

```abap
FUNCTION z_test.
  " You can use the template 'functionModuleParameter' to add here the signature!

  WRITE 'hello world'.
ENDFUNCTION.

```

When saving again, the comment is added once again:

```abap
FUNCTION Z_TEST
 " You can use the template 'functionModuleParameter' to add here the signature!
.

  " You can use the template 'functionModuleParameter' to add here the signature!

  WRITE 'hello world'.
ENDFUNCTION.
```

This can be repeated indefinitely.

These are the options I used:
![image](https://github.com/user-attachments/assets/ebb84c53-2574-4ac2-b7f1-af4980d1cf79)

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

very nice finding indeed! (And yes, the grammar in this auto-generated comment is painful!)

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

I could reproduce this issue on Aug. 22nd, but it seems to me that in the meantime, this comment is not generated anymore when the function module is saved! Nevertheless, I changed the cleanup rule to skip this exact case: 

![image](https://github.com/user-attachments/assets/dcf344f6-1f5a-4509-b50a-0edd123a05e8)

Kind regards,
Jörg-Michael

**Comment by ConjuringCoffee** [comment]
Thanks! 

I'm on the latest ADT version and the comment is still being generated for me. Maybe your ADT / your backend system is even more up-to-date 😉

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

thanks again for reporting this issue – this should now be fixed in version 1.19.3, which was just released! 

And yes, maybe I did have an insight into some future ADT version here, who knows ;-) 

Kind regards,
Jörg-Michael

**Comment by ConjuringCoffee** [comment]
Great, thanks :)
